src/easylink/steps/dev/build-containers-remote.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+set -e
+
+cd python_pandas
+singularity build --remote --force python_pandas.sif python_pandas.def
```

Comment:
Does `--remote` work for you on the cluster? Last I tried it, it didn't, but I like it!

Reply:
I haven't tried the script specifically but yeah that's how i built/tested the containers for the last PR, remotely on the cluster

Reply:
Nice! I didn't know we could do that.